packages/forms/resources/views/components/wizard.blade.php

```diff
@@ -10,7 +10,7 @@
         step: null,
 
         init: function () {
-            this.step = this.getSteps()[0]
+            this.step = this.getSteps()[{{ ($getStartOnStep() - 1) }}]
```

Comment:
I know this is nitpicky, but it's Alpine's responsibility to assign the step number correctly, not PHP

```suggestion
            this.step = this.getSteps()[{{ $getStartOnStep() }} - 1]
```